src/Umbraco.Core/Persistence/Dtos/DefaultConstraintPerColumnDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Core.Persistence.Dtos
+{
+    public class DefaultConstraintPerColumnDto
+    {
+        public string TableName { get; set; }
+        public string ColumnName { get; set; }
+        public string Name { get; set; }
+        public string Definition { get; set; }
```

Comment:
I think it would be great to have the other Dto classes using Pascal Case syntax as well.
Not sure the casing matter in the SQL query since it seems to use uppercase, but in that case it can probably be decorated with attributes as well.

Reply:
Some SQL queries already use aliases (`column AS newName` or `newName = column`), which can also be used to match the pascal casing.